packages/flutter/lib/src/cupertino/spell_check_suggestions_toolbar.dart

```diff
@@ -26,6 +30,18 @@ class CupertinoSpellCheckSuggestionsToolbar extends StatelessWidget {
     required this.buttonItems,
   });
 
+  /// Constructs a [CupertinoSpellCheckSuggestionsToolbar] with the default
+  /// children for an [EditableText].
+  ///
+  /// See also:
+  ///  * [SpellCheckSuggestionsToolbar.editableText], which is similar but
+  ///    builds an Android-style toolbar.
+  CupertinoSpellCheckSuggestionsToolbar.editableText({
```

Comment:
I'm curious -- with the `editableText` named constructors, do we need the other ones?

Reply:
* A `children`-based constructor - Could be useful for advanced customization.
  * A plain `editable` constructor - Could be useful for custom text editors.
  * `selectable` or `selectableRegion` - Unlikely to be useful because users probably don't want to spell check non-editable text?

I think let's wait and see if anyone opens some issues asking for these.  I would be up for it if they did though.